lib/registries/addon/drafts/draft/controller.ts

```diff
@@ -30,6 +31,38 @@ export default class RegistriesDrat extends Controller {
         return regex.test(this.page);
     }
 
+    @action
+    pageStateToClass(pageState: PageState): string {
+        switch (pageState) {
+        case PageState.Active:
+            return 'Active';
+        case PageState.Unvisited:
+            return 'Unvisited';
+        case PageState.Invalid:
+            return 'Invalid';
+        case PageState.Valid:
+            return 'Valid';
+        default:
+            return '';
+        }
+    }
+
```

Comment:
One more thing we could do (after moving this to the PageLinkManager component), instead of using the action to get a mapping, it might make sense to use a JS object for such mapping first, for example:
```
const PageClass = {
    [PageState.Active]: 'Active',
    [PageState.Unvisited]: 'Unvisited',
    ...
}
```
and then get the actual class using a computed, e.g.:
```
@computed(this.pageState)
get pageClass() {
    return PageClass[this.pageState];
}
```